src/algebra/monoid_algebra/basic.lean

```diff
@@ -684,6 +694,14 @@ lemma lift_unique (F : monoid_algebra k G →ₐ[k] A) (f : monoid_algebra k G)
   F f = f.sum (λ a b, b • F (single a 1)) :=
 by conv_lhs { rw lift_unique' F, simp [lift_apply] }
 
+/--  A multiplicative homomorphism `f : G →* H` between two monoids induces a `k`-algebra
+homomorphism `monoid_alg_hom_map k A f : monoid_algebra k G →ₐ[k] monoid_algebra k H`. -/
+def monoid_alg_hom_map (k A : Type*) [comm_semiring k] [semiring A] [algebra k A] {H : Type*}
```

Comment:
Shouldn't this also be named `map_domain_alg_hom`?

Reply:
I feel that I have renamed these lemmas dozens of times!  Thanks!